modules/Emails/EmailsController.php

```diff
@@ -200,14 +200,12 @@ public function action_send()
             );
 
             $this->view = 'ajax';
-            $response['errors'] = array(
+            $response['errors'] = [
```

Comment:
Not backwards compatible for PHP 5.5.9

Reply:
I think short array syntax was introduced in [5.4](http://php.net/manual/en/language.types.array.php#example-56)

Reply:
tested in 5.5.9: it's works for me